docs/alps/hardware.md

```diff
@@ -40,20 +40,36 @@ Alps was installed in phases, starting with the installation of 1024 AMD Rome du
 
 There are currently four node types in Alps, with another becoming available in 2025:
 
-| type           | blades | nodes | CPU sockets | GPU devices |
-| ----           | ------:| -----:| -----------:| -----------:|
-| NVIDIA GH200   | 1344   | 2688  | 10,752      | 10,752      |
-| AMD Rome       |  256   | 1024  |  2,048      | --          |
-| NVIDIA A100    |   72   |  144  |    144      | 576         |
-| AMD MI250x     |   12   |   24  |     24      |  96         |
-| AMD MI300A     |   64   |  128  |    512      | 512         |
+| type           | abbreviation  | blades | nodes | CPU sockets | GPU devices |
+| ----           | -------       | ------:| -----:| -----------:| -----------:|
+| NVIDIA GH200   | gh200         | 1344   | 2688  | 10,752      | 10,752      |
+| AMD Rome       | zen2          |  256   | 1024  |  2,048      | --          |
+| NVIDIA A100    | a100          |   72   |  144  |    144      | 576         |
+| AMD MI250x     | mi200         |   12   |   24  |     24      |  96         |
```

Comment:
Is `mi200` really the name we use for the MI250x or is it a typo?
```suggestion
| AMD MI250x     | mi250         |   12   |   24  |     24      |  96         |
```

Reply:
These are the names that we use to label uenv, and to tag runners, etc.
I can remove the column, but I think it is a good idea to have the same "tag" used for all services.